source/core/src/main/com/csse3200/game/components/lootboxview/LootBoxOverlayComponent.java

```diff
@@ -59,20 +60,32 @@ private void showLoot(List<AbstractItem> items) {
         // Clear the previous contents of the overlay
         overlayWindow.clearChildren();
 
-        // Add new items to the overlay
+        // Create a new table for displaying the loot items
         Table contentTable = new Table(skin);
+
+        // Set up a grid-like layout to prevent cramming, adjust columns based on items
+        int numColumns = 4;  // Adjust the number of columns to suit your layout
```

Comment:
I wonder if you could move these up into a private final int numColumns = 4, so it's easier to identify and change without having to go over the whole class.